src/pudl/package_data/ferc1/xbrl_factoid_plant_function_tags.csv

```diff
@@ -0,0 +1,30 @@
+﻿table_name,xbrl_factoid,plant_function
+plant_in_service_ferc1,intangible_plant,intangible_plant
+plant_in_service_ferc1,steam_production_plant,steam_production_plant
+plant_in_service_ferc1,nuclear_production_plant,nuclear_production_plant
+plant_in_service_ferc1,hydraulic_production_plant,hydraulic_production_plant
+plant_in_service_ferc1,other_production_plant,other_production_plant
+plant_in_service_ferc1,transmission_plant,transmission_plant
+plant_in_service_ferc1,distribution_plant,distribution_plant
+plant_in_service_ferc1,transmission_and_market_operation_plant_regional_transmission_and_market_operation_plant,transmission_and_market_operation_plant_regional_transmission_and_market_operation_plant
+plant_in_service_ferc1,general_plant,general_plant
+electric_operating_expenses_ferc1,steam_power_generation_operations_expense,steam_production_plant
+electric_operating_expenses_ferc1,steam_power_generation_maintenance_expense,steam_production_plant
+electric_operating_expenses_ferc1,nuclear_power_generation_operations_expense,nuclear_production_plant
+electric_operating_expenses_ferc1,nuclear_power_generation_maintenance_expense,nuclear_production_plant
+electric_operating_expenses_ferc1,hydraulic_power_generation_operations_expense,hydraulic_production_plant
+electric_operating_expenses_ferc1,hydraulic_power_generation_maintenance_expense,hydraulic_production_plant
+electric_operating_expenses_ferc1,other_power_generation_operations_expense,other_production_plant
+electric_operating_expenses_ferc1,other_power_generation_maintenance_expense,other_production_plant
+electric_operating_expenses_ferc2,other_power_supply_expense,other_power_supply
```

Comment:
Looks like there's some auto-incrementation of numbers for FERC1 happening here.